vault/core.go

```diff
@@ -482,60 +486,62 @@ type Core struct {
 
 // CoreConfig is used to parameterize a core
 type CoreConfig struct {
-	DevToken string `json:"dev_token" structs:"dev_token" mapstructure:"dev_token"`
```

Comment:
Just checking: Did you confirm these are not being used anywhere? Otherwise i'm not sure we should remove them

Reply:
Yep, I confirmed that with Jeff.  We are good to go on removing these.